ERCS/erc-7846.md

```diff
@@ -0,0 +1,233 @@
+--- 
+eip: 7846
+title: Wallet Connection API  
```

Comment:
I know there isn't a ton of room in the title (44 characters IIRC), but it would be nice if you could cram a bit more detail in here to help disambiguate.